packages/reference-apps/transform-sequence-1/package.json

```diff
@@ -19,5 +19,12 @@
   "repository": {
     "type": "git",
     "url": "https://github.com/scramjetorg/transform-hub.git"
+  },
+  "dependencies": {
+    "scramjet": "^4.35.12"
+  },
+  "publishConfig": {
```

Comment:
we are fully moving to npmjs, so this won't be necessary

Reply:
`dependencies` and `publishConfig`  removed in commit [ed25b4](https://github.com/scramjetorg/transform-hub/pull/12/commits/ed25b4f6c52ae573ffd937f3e9010644648c04d4)